packages/rs-dpp/src/data_contract/document_type/class_methods/create_document_types_from_document_schemas/mod.rs

```diff
@@ -72,35 +72,39 @@ impl DocumentType {
 }
 
 #[cfg(test)]
-mod test {
+mod tests {
+    use super::*;
+
+    use crate::consensus::basic::data_contract::DocumentTypesAreMissingError;
     use crate::consensus::basic::BasicError;
     use crate::consensus::ConsensusError;
+    use crate::data_contract::errors::DataContractError;
+    use assert_matches::assert_matches;
     use platform_value::Identifier;
     use std::ops::Deref;
 
     #[test]
     pub fn should_not_allow_creating_document_types_with_empty_schema() {
-        let result = crate::data_contract::document_type::DocumentType::create_document_types_from_document_schemas(
-            Identifier::random(),
+        let id = Identifier::random();
+
+        let result = DocumentType::create_document_types_from_document_schemas(
+            id,
             Default::default(),
             None,
             false,
             false,
             false,
             false,
             &mut vec![],
-            crate::version::PlatformVersion::latest(),
+            PlatformVersion::latest(),
         );
 
-        match result {
-            Err(crate::ProtocolError::ConsensusError(e)) => match e.deref() {
-                ConsensusError::BasicError(err) => match err {
-                    BasicError::DataContractEmptySchemaError(_) => {}
-                    _ => panic!("Expected DataContractEmptySchemaError"),
-                },
-                _ => panic!("Expected basic consensus error"),
-            },
-            _ => panic!("Expected consensus error"),
-        }
+        assert_matches!(result, Err(ProtocolError::ConsensusError(e)) => {
```

Comment:
Actually, vice versa. When you use `assert!(matches!(...))` you don't see the actual value what you are trying to match in case of error which is very inconvenient. That's why I introduced `assert_matches`